community/security/src/main/java/org/neo4j/server/security/auth/AuthProcedures.java

```diff
@@ -64,20 +64,21 @@ public void deleteUser( @Name( "username" ) String username ) throws InvalidArgu
     }
 
     @Deprecated
-    @Description( "Change the user password. Deprecated by dbms.security.changePassword." )
+    @Description( "Change the current users password. Deprecated by dbms.security.changePassword." )
```

Comment:
Should this (and occurences below) be "user's password", or is it common to avoid apostrophe in possessive form?